packages/interaction/src/InteractionManager.ts

```diff
@@ -1027,8 +1076,12 @@ export class InteractionManager extends EventEmitter
 
             displayObject.emit(eventString, eventData);
 
-            if (displayObject[eventString])
+            // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+            // @ts-ignore
+            if (displayObject[eventString] && displayObject[eventString] instanceof Function)
             {
+                // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+                // @ts-ignore
                 displayObject[eventString](eventData);
```

Comment:
This can be done with 
```javascript
 const event = Object.entries(displayObject).find((entry) => entry[0] === eventString);
```
instead of `@ts-ignore` but I don't think it's a good way since it'd take more time to fix just a type error

Reply:
The problem with `Object.entries` and `Array.prototype.find` is that these are not supported across all platforms. We need to support IE and older browsers, some that don't support these ES6 features. In general, we have a polyfills packages that contains some things, but for some Object or Array features, we don't bother with polyfilling or transpiling and simply implement in an ES5 way.